index.bs

```diff
@@ -691,6 +745,32 @@ run these steps:
 6. Map |intersectionRect| to the coordinate space of the <a>viewport</a> of the {{document}} containing |target|.
 7. Return |intersectionRect|.
 
+<h4 id='calculate-visibility-algo'>
+Compute whether a Target is unoccluded, untransformed, unfiltered, and opaque.</h4>
+
+To compute the <dfn>visibility</dfn> of a <a for="IntersectionObserver">target</a>, run these steps:
+1. If the |observer|'s {{IntersectionObserver/trackVisibility}} attribute is false, return true.
```

Comment:
return `false`? see comment about line 468

Reply:
Agreed, I will change this.